src/Microsoft.Build.Sql.Templates/sqlproject/SqlProject1.sqlproj

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build">
+  <Sdk Name="Microsoft.Build.Sql" Version="0.1.3-preview" />
```

Comment:
We should match the templates version to the SDK version, or always use the latest available SDK version? I can send a PR to set the version at build time after this is merged.

Reply:
until GA it would be latest in preview, I'm good with that being set at build time